### PR TITLE
[TASK] Revise chapter "TYPO3_CONF_VARS"

### PR DESCRIPTION
**Pull request by brotkrueml**
Releases: main, 11.5

### CODE REVIEW

File: Documentation/Configuration/Typo3ConfVars/Index.rst [hunk 1]
```diff
@@ -11,8 +11,8 @@ TYPO3_CONF_VARS
 The main configuration is achieved via a set of global settings
 stored in a global array called :php:`$GLOBALS['TYPO3_CONF_VARS']`.
 
-This chapter describes this global configuration in more details and hints
-at other configuration possibilities.
+This chapter describes this global configuration in more details and give hints
```
Comment (linawolf) [inline Documentation/Configuration/Typo3ConfVars/Index.rst:14]:
```suggestion
This chapter describes this global configuration in more details and give hints
```
```suggestion
This chapter describes this global configuration in more details and gives hints
```
gives hints or hints

File: Documentation/Configuration/Typo3ConfVars/Index.rst [hunk 2]
```diff
@@ -148,54 +148,55 @@ may themselves be arrays.
...
-   and the backend.
+    :ref:`Service registration configuration <services-configuration-registration-changes>`
+    and the backend.
 
 Further details on the various configuration options can be found in the
-:guilabel:`Admin Tools` as well as the TYPO3 source at
-:file:`typo3/sysext/core/Configuration/DefaultConfigurationDescription.yaml`.
-The documentation shown in the :guilabel:`Admin Tools` is automatically
+:guilabel:`Admin Tools` module as well as the TYPO3 source at
+:file:`EXT:core/Configuration/DefaultConfigurationDescription.yaml`.
```
Comment (linawolf) [inline Documentation/Configuration/Typo3ConfVars/Index.rst:194]: shall we link this on git?

Comment (brotkrueml) [reply Documentation/Configuration/Typo3ConfVars/Index.rst:194]: Nope, as when we link it there we have to link to a dedicated branch (or commit) - which will be wrong in some doc branches.

Comment (linawolf) [reply Documentation/Configuration/Typo3ConfVars/Index.rst:194]: I think linking such files invites readers to have a look at the code. Expecially now that the code is harder to find for newbies with the different positions. We could define a link like :github: that links to different versions according to the settings.cfg

Comment (brotkrueml) [reply Documentation/Configuration/Typo3ConfVars/Index.rst:194]: Okay, when this works, go for it. But this is not part of this PR.

**Review by linawolf — APPROVED** [review]

**Pull request merged**